PowerEditor/src/ScintillaComponent/columnEditor.rc

```diff
@@ -33,19 +33,20 @@ BEGIN
 
     GROUPBOX        "",IDC_COL_NUM_GRP_STATIC,8,77,204,130
     RTEXT           "&Initial number:",IDC_COL_INITNUM_STATIC,10,89,76,8
-    EDITTEXT        IDC_COL_INITNUM_EDIT,90,87,38,12,ES_NUMBER
+    EDITTEXT        IDC_COL_INITNUM_EDIT,90,87,38,12,ES_LEFT
     RTEXT           "Increase b&y:",IDC_COL_INCRNUM_STATIC,10,106,75,8
-    EDITTEXT        IDC_COL_INCREASENUM_EDIT,90,104,38,12,ES_NUMBER
+    EDITTEXT        IDC_COL_INCREASENUM_EDIT,90,104,38,12,ES_LEFT
     RTEXT           "&Repeat:",IDC_COL_REPEATNUM_STATIC,10,123,75,8
-    EDITTEXT        IDC_COL_REPEATNUM_EDIT,90,121,38,12,ES_NUMBER
+    EDITTEXT        IDC_COL_REPEATNUM_EDIT,90,121,38,12,ES_LEFT
     RTEXT           "&Leading:",IDC_COL_LEADING_STATIC,10,140,75,8
     COMBOBOX        IDC_COL_LEADING_COMBO,90,138,100,30,CBS_DROPDOWNLIST | WS_TABSTOP
 
     GROUPBOX        "Format",IDC_COL_FORMAT_GRP_STATIC,16,155,188,44,BS_CENTER
-    CONTROL         "&Dec",IDC_COL_DEC_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,27,168,70,10
-    CONTROL         "&Hex",IDC_COL_HEX_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,122,168,70,10
-    CONTROL         "&Oct",IDC_COL_OCT_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,27,183,70,10
-    CONTROL         "&Bin",IDC_COL_BIN_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,122,183,70,10
+    CONTROL         "&Dec",IDC_COL_DEC_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,27,168,50,10
+    CONTROL         "&Hex",IDC_COL_HEX_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,110,168,50,10
+    CONTROL         "&Oct",IDC_COL_OCT_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,27,183,50,10
+    CONTROL         "&Bin",IDC_COL_BIN_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,110,183,50,10
+    COMBOBOX        IDC_COL_HEXUC_COMBO,150,168,40,10,CBS_DROPDOWNLIST | WS_TABSTOP

```

Comment:
I would use `IDC_COL_HEXUC_COMBO,150,166,40,10,CBS_DROPDOWNLIST | WS_TABSTOP` for better horizontal alignment with hex radio button